packages/dendron-plugin-views/src/components/DendronApp.tsx

```diff
@@ -115,6 +115,14 @@ function DendronApp(props: DendronAppProps) {
   const opts = _.defaults(props.opts, {
     padding: props.Component.name === "DendronGraphPanel" ? "0px" : "33px",
   });
+
+  useEffect(() => {
+    // this class is introduced with new reduced Motion feature of vscode and is resulting
+    // in a bug where webviews are updated to light theme.
+    //More info about this class here: https://code.visualstudio.com/api/extension-guides/webview#accessibility
+    document.body.classList.remove("vscode-reduce-motion");
```

Comment:
i'm fine with doing the quickfix - do you know what the underlying issue is that is causing this class to result in a light theme? can we create a followup task with details and hypothesis on how to properly address it?

Reply:
Sure I'll add a follow-up task for investigating the underlying issue.

Reply:
Added a new commit for the fix. The underlying issue was how we set the dendron theme.